firebase.json

```
@@ -438,6 +443,11 @@
           "destination": "/docs/tools/hardhat/hardhat-zksync-verify.html",
           "type": "302"
         },
+        {
+          "source": "/docs/tools/hardhat/testing.html",
+          "destination": "/docs/tools/local-testing/",
+          "type": "302"
```

```suggestion
          "type": "301"
```
Let's do 301 redirects as those are permanent